src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/YesNo.java

```diff
@@ -32,15 +33,15 @@ public static boolean isYes(YesNo yesNo) {
     }
 
     public static boolean isYes(String yesNo) {
-        return YES.getValue().equals(yesNo);
+        return nonNull(yesNo) && YES.getValue().equalsIgnoreCase(yesNo);
     }
 
     public static boolean isNoOrNull(YesNo yesNo) {
         return isNull(yesNo) || NO.equals(yesNo);
     }
 
     public static boolean isNoOrNull(String yesNo) {
-        return isNull(yesNo) || NO.getValue().equals(yesNo);
+        return isNull(yesNo) || NO.getValue().equalsIgnoreCase(yesNo);
     }
```

Comment:
Why are we not simply doing Boolean.parseBoolean(yesNo) ?
The YesNo class just seems to add code for the sake of it.
NB: For the sake of time, don't go changing this, but just needed to raise this question somewhere :)

Reply:
Boolean.parseBoolean(yesNo) , if the data is "yes" will return false

Reply:
My bad, I meant this:
BooleanUtils.toBoolean(String)

It's an apache commons class.